apps/web-giddh/src/app/services/thermal.service.ts

```diff
@@ -0,0 +1,730 @@
+
+import { Injectable } from '@angular/core';
+import { PrinterFormatService } from './printer.format.service';
+import * as qz from "qz-tray";
+
+@Injectable()
+export class ThermalService {
+
+    /** This will use for max length for character according to paper */
+    private maxLength: number = 46;
+
+    constructor(private printerFormat: PrinterFormatService) { }
+
+    /**
+    * This will use for pos commands formatted
+    *
+    * @memberof ThermalComponent
+    */
+    public print(defaultTemplate: any, request: any): void {
+
+        this.maxLength = 46;
+        this.maxLength = +this.maxLength;
```

Comment:
This statement is of no use as on line 21 `this.maxLength` is assigned a numeric value